rosbag2_test_common/package.xml

```diff
@@ -9,12 +9,11 @@
 
   <buildtool_depend>ament_cmake</buildtool_depend>
 
-  <test_depend>ament_cmake_gmock</test_depend>
+  <build_export_depend>rclcpp</build_export_depend>
+  <build_export_depend>rcutils</build_export_depend>
```

Comment:
The package needs both `rclcpp` and `rcutils` at build time (since it calls `find_package()` for them). So these need to be `build_depend` as well a run dependency - since you also need their shared libraries at runtime I would use `exec_depend` instead of `build_export_depend`.

Reply:
done in https://github.com/ros2/rosbag2/pull/118/commits/4a5ddc61044b404f21de5d58344b4a9969e969ed